lib/plivo/resources/brand.rb

```diff
@@ -76,6 +76,15 @@ def create(options=nil)
         end
         perform_create(options)
       end
+
+        ##
+        # Get BrandUsecase
+        # @param [String] brand_id
+      def get_usecases(brand_id)
+        valid_param?(:brand_id, brand_id, [String, Symbol], true)
+        identifier = brand_id + '/usecases'
+        perform_action_with_identifier(identifier, 'GET', nil)
```

Comment:
what is the type of output here? Object or Json. I can see there is new fucntion added to_json_brand_usecase.

Reply:
It returns an object here, but in UT we need to compare jsons. Hence added the function in spec/resource_brand_spec.rb

Reply:
other brand functions also returning object? why different behaviour?

Reply:
yes, other functions are also returning response object